dencam/gui.py

```diff
@@ -103,14 +103,9 @@ def _update(self):
 
         self.recorder.update_timestamp()
 
-        if self.state.value == 1:
-            self.show_frame('NetworkPage')
-        elif self.state.value == 2:
-            self.show_frame('RecordingPage')
-        elif self.state.value == 3:
-            self.show_frame('SolarPage')
-        elif self.state.value == 4:
-            self.show_frame('BlankPage')
+        if (self.state.value >= self.STATE_LIST.index('NetworkPage')
+              and self.state.value <= self.STATE_LIST.index('BlankPage')):
```

Comment:
This is a good idea.  Removes more magic numbers.  (Although does assume the order of the pages but let's not worry about that.)

However, this pair of lines doesn't pass my linter because of the indenting (did it pass yours?).  The following does:

```
if(self.state.value >= self.STATE_LIST.index('NetworkPage')
   and self.state.value <= self.STATE_LIST.index('BlankPage')):
```